pkgs/by-name/ze/zed-editor/package.nix

```
@@ -301,14 +301,11 @@ rustPlatform.buildRustPackage (finalAttrs: {
     versionCheckHook
   ];
   versionCheckProgram = "${placeholder "out"}/bin/zeditor";
-  versionCheckProgramArg = [ "--version" ];
+  versionCheckProgramArg = "--version";
   doInstallCheck = true;
 
   passthru = {
-    updateScript = gitUpdater {
-      rev-prefix = "v";
-      ignoredVersions = "(*-pre|0.999999.0|0.9999-temporary)";
-    };
+    updateScript = nix-update-script { };
```

This should resolve https://github.com/NixOS/nixpkgs/pull/387928/files#r1985226790 :)

```suggestion
    updateScript = nix-update-script {
      extraArgs = [
        "--version-regex"
        "(?!.*(?:-pre|0\.999999\.0|0\.9999-temporary)$)(.+)"
      ];
    };
```